keyboards/park-csu/lyra/info.json

```diff
@@ -0,0 +1,118 @@
+{
+    "keyboard_name": "lyra",
+    "url": "",
+    "maintainer": "Changsu Park",
+    "width": 15,
+    "height": 4,
```

Comment:
```suggestion
```

Reply:
This is essencially same right?

Reply:
specifically, these values are not needed, and are automatically generated.  So they're deprecated.